src/panels/config/entities/ha-config-entities.ts

```diff
@@ -624,8 +632,10 @@ export class HaConfigEntities extends SubscribeMixin(LitElement) {
         const entity = this.hass.states[entry.entity_id];
         const unavailable = entity?.state === UNAVAILABLE;
         const restored = entity?.attributes.restored === true;
-        const areaId = entry.area_id ?? devices[entry.device_id!]?.area_id;
+        const deviceId = entry.device_id;
+        const areaId = entry.area_id ?? devices[deviceId!]?.area_id;
```

Comment:
```suggestion
        const areaId = entry.area_id || devices[deviceId!]?.area_id;
```

I think `area_id` is a string. `??` would just make sense if it's an integer and it can be `0`